cmd/create/project.go

```diff
@@ -31,65 +31,34 @@ Create a project by definition file. Note: The name shouldn't contain any whites
     id: "project-unique-id"
     name: "Name"
     labels:
-     app: flyte
+       values:
+         app: flyte
     description: "Some description for the project"
```

Comment:
@EngHabu This change is not backward compatibly, Is it ok to change the file structure

Reply:
It's ok to break the backward compatibility